test/common_interface/ida.jl

```diff
@@ -7,38 +7,49 @@ prob = prob_dae_resrob
 dt = 1000
 saveat = float(collect(0:dt:100000))
 sol = solve(prob,IDA())
+@info "Multiple abstol"
 sol = solve(prob,IDA(),abstol=[1e-9,1e-8,1e-7])
+@info "Band solver"
 sol2 = solve(prob,IDA(linear_solver=:Band,jac_upper=2,jac_lower=2))
+@info "GMRES solver"
 sol3 = solve(prob,IDA(linear_solver=:GMRES))
 #sol4 = solve(prob,IDA(linear_solver=:BCG)) # Fails but doesn't throw an error?
+@info "TFQMR solver"
 sol5 = solve(prob,IDA(linear_solver=:TFQMR))
+@info "FGMRES solver"
 sol6 = solve(prob,IDA(linear_solver=:FGMRES))
+@info "PCG solver"
 sol7 = solve(prob,IDA(linear_solver=:PCG)) # Requires symmetric linear
+#@info "KLU solver"
+#sol8 = solve(prob,IDA(linear_solver=:KLU)) # Requires Jacobian
```

Comment:
so KLU isn't building correctly?

Reply:
If it is an issue with how SuiteSparse is built, please ping me.

Reply:
At least for the Sundials 3 binaries, I faced issues with the Jacobian tests.

Reply:
With the Sundials 5 binaries, they probably segfault without more work to fix function signatures like:

https://github.com/SciML/Sundials.jl/blob/master/src/common_interface/function_types.jl#L42-L55 
https://github.com/SciML/Sundials.jl/blob/master/src/common_interface/function_types.jl#L57-L64

since it changed a lot of those around. So since @jd-lara isn't doing the common interface here, any segfaults we get right now would have confounding factors. @YingboMa and I will dig into this in subsequent PRs, but IIRC this Sundials 5 version is 64-bit LAPACK compatible, so the issue with the Sundials 3 binaries shouldn't be in here anymore. Only time will tell.

Reply:
That's pretty much where I was seeing the failures.

Reply:
You were also seeing failures in the Jacobians calls too though, this is just sparse Jacobians.